includes/Data/Themes.php

```diff
@@ -112,7 +112,7 @@ public static function get_init() {
 	}
 
 	// This is temporary, as we implement theme selections we can remove this.
-	public static function get_flow_default_theme_slug( $flow ) {
+	protected static function get_flow_default_theme_slug( $flow ) {
```

Comment:
The function is also used outside this class in `ThemeGeneratorController` that does not extend this class, pretty sure this is also a fatal error.